src/twisted/web/newsfragments/.gitignore

```diff
@@ -0,0 +1 @@
+!.gitignore
```

Comment:
It would seem that this would be more clearly expressed by just having a file called `.exists-for-towncrier` in each directory?